test/validate/eia_test.py

```diff
@@ -110,83 +110,63 @@ def test_minmax_rows(
     [
         (
             "bf_eia923",
-            (
-                Package.from_resource_ids()
-                .get_resource("boiler_fuel_eia923")
-                .schema.primary_key
-            ),
+            Package.from_resource_ids()
+            .get_resource("boiler_fuel_eia923")
+            .schema.primary_key,
```

Comment:
It probably doesn't matter much in terms of speed, but I think that these could all be replaced with e.g.

```py
Resource.from_id("boiler_fuel_eia923").schema.primary_key
```